tools/licenses/README.md

```diff
@@ -20,9 +29,26 @@ rm -rf ../../../out/licenses
 dart lib/main.dart --src ../../.. --out ../../../out/licenses --golden ../../ci/licenses_golden
 ```
 
-Then copy any affected files from `../../../out/licenses` to `../../ci/licenses_golden` and add them to your change.
+Then, copy any affected files from `../../../out/licenses` to
+`../../ci/licenses_golden` and add them to your change, and examine
+the diffs to see what changed.
+
+```
+cp ../../../out/licenses/* ../../ci/licenses_golden
+git diff
+```
+
+If the only changes are to what files are included, then you're good
+to go. However, if any of the _licenses_ changed, whether new licenses
+are added, old ones removed, or any have their content changed in any
+way, you must update the actual license file.
+
+## IF ANYTHING SIGNIFICANT CHANGED, YOU MUST UPDATE THE ACTUAL LICENSE FILE AS WELL
```

Comment:
What constitutes "significant"?  Best if that's not left open to interpretation.

Reply:
it's defined in the paragraph above :-)

Reply:
i removed this header to make it clearer

Reply:
OK, I guess, but it still feels like leaving it open for interpretation. Maybe this?:
```suggestion
## IF ANY LICENSES CHANGED, YOU MUST UPDATE THE ACTUAL LICENSE FILE AS WELL
```